backend/chunker/chunker.go

```diff
@@ -2139,9 +2179,10 @@ func (o *Object) ID() string {
 // Meta format `simplejson`
 type metaSimpleJSON struct {
 	// required core fields
-	Version  *int   `json:"ver"`
-	Size     *int64 `json:"size"`    // total size of data chunks
-	ChunkNum *int   `json:"nchunks"` // number of data chunks
+	Version  *int    `json:"ver"`
+	Size     *int64  `json:"size"`    // total size of data chunks
+	ChunkNum *int    `json:"nchunks"` // number of data chunks
+	XactID   *string `json:"xactID"`  // transaction ID for a completed upload
```

Comment:
Please mark as `omitempty` and put after the _optional_ comment.
This will help to keep backwards compatibility.

Reply:
I believe this is the small fix you mentioned.